lib/fork.js

```diff
@@ -7,6 +7,21 @@ var debug = require('debug')('ava');
 var AvaError = require('./ava-error');
 var send = require('./send');
 
+var env = process.env;
+
+if (process.env.NODE_PATH) {
+	var nodePath = process.env.NODE_PATH
+		.split(path.delimiter)
+		.map(function (p) {
+			return path.resolve(p);
+		})
+		.join(path.delimiter);
+
+	env = objectAssign({
+		NODE_PATH: nodePath
+	}, env);
```

Comment:
Ugh, discovered this too late. This makes no sense. Here you overwrite the modified `NODE_PATH` with the original one. This means the tests are also invalid. Tests still pass when the `.map` is commented out.

// @ingro @novemberborn